am-lib/src/main/java/uk/gov/hmcts/reform/amlib/models/RoleBasedAccessGrant.java

```diff
@@ -0,0 +1,15 @@
+package uk.gov.hmcts.reform.amlib.models;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@Builder
+@AllArgsConstructor
+public class RoleBasedAccessGrant {
```

Comment:
I do not see why we need this. No Javadoc to help me understand.

Reply:
Another pmd ask. Will add documentation now.

Reply:
SURPRESS THIS